apps/job-service/src/jobs/delayed-jobs.controller.ts

```diff
@@ -90,25 +96,34 @@ export class DelayedJobsController {
         uuid: { [Op.in]: jobUpdates.map(({ uuid }) => uuid) },
         createdBy: authenticatedUserId,
         status: { [Op.ne]: "pending" }
-      }
+      },
+      logging: console.log,
+      order: [["createdAt", "DESC"]]
     });
-    if (jobs.length !== jobUpdates.length) {
+
+    if (!jobs.length) {
```

Comment:
I'm curious about this change; shouldn't this still compare against the length of jobs in the request body?

Reply:
Hmm, but this code as written won't detect if the FE tried to update a job that the user doesn't own. It would be better for the FE to only send the IDs that it expects to be able to update (so, filtering out pending ones) so that the BE can raise the issue correctly. I would certainly consider it an error that the BE should complain about if the FE asks it to update job that can't be updated (because it's in pending, or because it's owned by another user doesn't make much difference; it's a breaking in the contract of the endpoint to not update something that was asked to be updated and also not complain about it)

Reply:
this has being changed. Now The FE will only send the jobs that will be updated.

Reply:
reverted the change too